plugins/org.locationtech.udig.catalog.db2/src/org/locationtech/udig/catalog/db2/internal/messages.properties

```diff
@@ -15,5 +15,7 @@ DB2WizardPage_installDrivers=Install DB2 drivers from client
 DB2WizardPage_connectionTask=Attempting to make database connection
 DB2WizardPage_warning=An error will occur because the DB2 drivers were not loaded.  Please restart application
 DB2WizardPage_title=DB2 Wizard
+DB2WizardPage_button_lookup_text=Get schemas
```

Comment:
The screenshot needs to be changed - how should it be provided?

The "Adding features from ..." doesn't seem useful for ArcSDE, DB2 or Oracle - it is basically the shapefile instructions. It is correct for PostGIS.
How can I update this information for DB2?

There is no change to the preferences page.